src/topics/lda_model.cpp

```diff
@@ -4,6 +4,7 @@
  */
 
 #include "meta/topics/lda_model.h"
+#include <iostream>
```

Comment:
Is this for `std::ofstream`? That would be `#include <fstream>`. Also, I like to put system includes above the meta includes, but that's so nitpicky...